site/docs/extensions/filter-control.md

```diff
@@ -57,18 +55,6 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 - **Default:** `undefined`
 
-### disableControlWhenSearch
```

Comment:
Why do you removed that option ?

Reply:
Well, it wasn't working and I think this can be done by the developer

Reply:
Okay, i just wanted to know the reason.
I agree :-)